spacelift/resource_bitbucket_datacenter_integration.go

```diff
@@ -19,39 +28,35 @@ func resourceBitbucketDatacenterIntegration() *schema.Resource {
 		UpdateContext: resourceBitbucketDatacenterIntegrationUpdate,
 		DeleteContext: resourceBitbucketDatacenterIntegrationDelete,
 
-		Importer: &schema.ResourceImporter{
```

Comment:
@truszkowski Hello Piotr, just seen this change. You removed this default importer from the resource, and it doesn't support importing existing resources into state file anymore. Was that a conscious decision or can we add it back?

Reply:
I think we just missed it. I saw you put a new PR related to this. I'll take care of this.
Big thanks ❤️!